### PR TITLE
Feature cucumber group addmoretest

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/test/java/salesforce/steps/groupGeneralSteps.java [hunk 1]
```diff
@@ -80,16 +79,27 @@ public void iUseEndpointRequestToObjectWithName(String endpoint, String name, St
...
         apiResponse = ApiManager.execute(apiRequest);
     }
 
     @Then("the response status code should be {string} to group")
     public void theResponseStatusCodeShouldBeToProduct(String statusCode) {
-        log.info("I verify status response");
+        log.info("I verify status response of group");
         Assert.assertEquals(apiResponse.getStatusCode(), HttpStatus.valueOf(statusCode).value());
         apiResponse.getResponse().then().log().body();
     }
```
Comment (jpgonzales27) [inline src/test/java/salesforce/steps/groupGeneralSteps.java:92]: Maybe this line should be separated

File: src/test/resources/features/group.feature [hunk 2]
```diff
@@ -41,3 +79,20 @@ Feature: Endpoint group
...
+    Examples:
+      | parameterToUpdate     | updateDate        |
+      |                       | change name group |
+      | name                  |                   |
+      | visibility            | null              |
+      | visibility            | Public            |
+      | isArchived            |                   |
+      | isAutoArchiveDisabled |                   |
+      | isBroadcast           |                   |
+      | canHaveChatterGuests  | true              |
```
Comment (jpgonzales27) [inline src/test/resources/features/group.feature]: Should be to add one line at the end of the file

File: src/test/java/salesforce/steps/groupGeneralSteps.java (hunk 1 above)
Comment (GustavoZHA) [reply src/test/java/salesforce/steps/groupGeneralSteps.java:92]: I forgot

File: src/test/resources/features/group.feature (hunk 2 above)
Comment (GustavoZHA) [reply src/test/resources/features/group.feature]: ok

**Pull request merged**